src/me/StevenLawson/TotalFreedomMod/Commands/Command_spectator.java

```diff
@@ -0,0 +1,73 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import me.StevenLawson.TotalFreedomMod.TFM_AdminList;
+import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import org.bukkit.GameMode;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.OP, source = SourceType.BOTH)
```

Comment:
@Wild1145 @xDestroyer217 Since spectators are also invisible, I think that only admins should be able to use this command.